### ISSUE TITLE
POST request bug

### ISSUE THREAD

**Issue by bhawnishk** [daniely93/nodejs-server-maintenance#4]
Hi there, it seems like the POST request; http://localhost:3000/maintenance?access_key=**** doesn't seem to turn maintenance mode ON or off. Also my another feedback, under "Setting maintenance mode on" section in the readme, can you explicitly specify /maintenance as param instead of /[endpoint] which is kind of ambiguous as I was trying it as http://localhost:3000/localhost:3000/roles?access_key=*** when it should've been  http://localhost:3000/maintenance?access_key=****, so I think you can replace it with "POST request to http://yourendpoint/maintenance?access_key=[accessKey]
and same for Delete request below.

**Comment by daniely-93** [comment]
Hi,

Please read the note:

> The maintenance should be defined before your app routes and after your body parser

Did you use `app.use(bodyParser)`? if you did not, your app will throw this error 

> TypeError: Cannot convert undefined or null to object 

Here is an example so you can test:

```
const express = require("express");
const app = express();
const maintenance = require("nodejs-server-maintenance");
const port = 3000;

const jsonParser = express.json();
app.use(jsonParser);

const options = {
  mode: false,
  accessKey: "1234",
  endpoint: "/maintenance123",
  filePath: null,
  useApi: false,
  statusCode: 503,
  message:
    "Error 503: Server is temporarily unavailable, please try again lager.",
  blockMethods: ["GET"],
};

maintenance(app, options);

app.get("/test", (req, res) => {
  res.send("Hello!");
});

app.listen(port, () => console.log(`Server is running on port ${port}`));

```

As you can see, the endpoint is '/maintenance123' and the access key is '1234'
POST request to 

> http://localhost:3000/maintenance123?access_key=1234

Will give you the response
```
{
    "success": true,
    "mode": true
}
```

And if you try GET request to 

> http://localhost:3000/test

it will response with the maintenance mode on.

**Comment by bhawnishk** [comment]
@daniely-93 The error is showing up now, but the statusCode is not changing at all it keeps returning 200 OK when it should be returning 503: 
![4690FDE5-A0C4-48D7-9B4A-88AC3E855ADF](https://user-images.githubusercontent.com/91909201/184396584-aa9fb5b4-5628-48f6-b648-339053a24dc1.jpeg)


I even tried changing it with the POST request by sending { statusCode: 503, message: "etc..etc.." }, the message did change however the statusCode is still 200 OK:
![Image 8-12-22 at 9 02 AM](https://user-images.githubusercontent.com/91909201/184396957-0339afd2-098a-464e-8fe5-aa42d61b41bb.jpg)

**Comment by daniely-93** [comment]
Fixed on version 2.1.1.
When the maintenance mode is on, it will send the status code
![image](https://user-images.githubusercontent.com/56507321/184507378-40851f4c-d7ee-4664-b714-63159ab82ede.png)
